src/locutus/model/terminology.py

```diff
@@ -276,7 +290,16 @@ def rename_code(
                     return True
         return False
 
-    def delete_mappings(self, editor, code=None):
+    def soft_delete_mappings(self, editor, code=None):

```

Comment:
I mentioned in previous comments that my personal preference would be that soft deletes vs regular deletes are an implementation detail that doesn't need to be described with the function name. However, the comments below are 100% appropriate.